cardano-config/src/Cardano/Config/ShelleyGenesis.hs

```diff
@@ -257,5 +261,5 @@ serialiseAddr = serialize'
     -- the utils are ready and so we will not forget.
       _unused = mkRwdAcnt
 
-deserialiseAddr :: Crypto crypto => ByteString -> Maybe (Addr crypto)
-deserialiseAddr = either (const Nothing) id . decodeFull'
+deserialiseAddr :: Crypto crypto => ByteString -> Either DecoderError (Addr crypto)
+deserialiseAddr = decodeFull'
```

Comment:
Don't change the type here. This is the exact type that the upstream functions provide. If you change it here it'll just cause a type error once we update and the upstream utils become available.

Reply:
@dcoutts there is an issue here: https://input-output-rnd.slack.com/archives/G011N23CEAE/p1587094591064500
I'll need some time to fix this another way.